src/openforms/submissions/tasks/emails.py

```diff
@@ -133,6 +140,12 @@ def send_email_cosigner(submission_id: int) -> None:
                 from_email=settings.DEFAULT_FROM_EMAIL,
                 recipient_list=[recipient],
                 text_message=content,
+                extra_headers={
+                    "Content-Language": submission.language_code,
+                    X_OF_CONTENT_TYPE_HEADER: EmailContentTypeChoices.submission,
+                    X_OF_CONTENT_UUID_HEADER: str(submission.uuid),
+                    X_OF_EVENT_HEADER: EmailEventChoices.cosign_confirmation,
```

Comment:
This email isn't a cosign confirmation email, but rather a cosign request. To be fair, I think the original code defining the event is wrong and should be renamed to 'cosign-request', since the cosign confirmation is not much different from the "normal" confirmation email (if I remember correctly).